ios/Helio.xcodeproj/xcshareddata/xcschemes/HelioDev.xcscheme

```diff
@@ -79,14 +78,14 @@
             ActionType = "Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction">
             <ActionContent
                title = "Run Script"
-               scriptText = "echo &quot;.env.development&quot; &gt; /tmp/envfile&#10;">
+               scriptText = "# Type a script or drag a script file from your workspace to insert its path.&#10;# Type a script or drag a script file from your workspace to insert its path.&#10;echo &quot;.env.development&quot; &gt; /tmp/envfile&#10;">
                <EnvironmentBuildable>
                   <BuildableReference
                      BuildableIdentifier = "primary"
                      BlueprintIdentifier = "889CAB5E27D46CD6007F5892"
-                     BuildableName = "BetterSocialDev.app"
-                     BlueprintName = "BetterSocialDev"
-                     ReferencedContainer = "container:BetterSocial.xcodeproj">
+                     BuildableName = "HelioDev.app"
+                     BlueprintName = "HelioDev"
+                     ReferencedContainer = "container:Helio.xcodeproj">
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The script text for the Run Script action has been modified. It now includes two comment lines. Make sure this change is intended as it could affect the build process.
```diff
- scriptText = "echo &quot;.env.development&quot; &gt; /tmp/envfile&#10;"
+ scriptText = "# Type a script or drag a script file from your workspace to insert its path.&#10;# Type a script or drag a script file from your workspace to insert its path.&#10;echo &quot;.env.development&quot; &gt; /tmp/envfile&#10;"
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->